services/datamanager/data/fake_robot_with_data_manager.json

```diff
@@ -15,7 +15,7 @@
                         "capture_methods": [
                             {
                                 "method": "EndPosition",
-                                "capture_frequency_hz": 500,
+                                "capture_frequency_hz": 100,
```

Comment:
Made these capture rates the same to simplify the mockClock.Add(captureInterval) logic (only needing a single value for the captureInterval)